cmd/rtrmon/rtrmon.go

```diff
@@ -358,9 +358,9 @@ func (c *Client) Start(id int, ch chan int) {
 				tmpVrpMap[key] = &vrpSimple
 			}
 			c.compLock.Lock()
+			defer c.compLock.Unlock()
```

Comment:
will remove this one because it will still hold the lock while communicating over the channel..